config/routes.rb

```diff
@@ -8,7 +8,7 @@
   get '/profile', to: 'users#show'
 
   get '/autocomplete', to: 'google/autocomplete#index'
-  get '/map', to: 'static#map'
+  get '/search', to: 'google/search#index'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.